config/kubernetes-nightly/org.yaml

```diff
@@ -0,0 +1,42 @@
+name: Kubernetes Nightly
+description: Org for Kubernetes publishing-bot work
+default_repository_permission: read
+has_organization_projects: true
+has_repository_projects: true
+members_can_create_repositories: false
+billing_email: github@kubernetes.io
+admins:
```

Comment:
One question @justaugustus -- if there are teams that have access to the repos, do we need to have all the release managers as org admins here too? Wondering if there is a technical reason :)

Reply:
> One question @justaugustus -- if there are teams that have access to the repos, do we need to have all the release managers as org admins here too? Wondering if there is a technical reason :)

Currently, only members of the @kubernetes/owners + @kubernetes/publishing-bot-admins (pub-bot approvers + @kubernetes/sig-release-admins) are listed as org admins.

Our default repo creation permission is:

```yaml
members_can_create_repositories: false
```

Given the `kubernetes-nightly` org exists to test the creation of repos, some subset of SIG Release needs access to do so.

As we learn more (and get the process documented), we can probably start to:

- automate more (bot account + GCB job (or something))
- revisit org admins

Reply:
Sounds great! Thanks @justaugustus !